servicex/models.py

```diff
@@ -34,6 +34,8 @@
 class TransformRequest(db.Model):
     __tablename__ = 'requests'
 
+    _cache = {}
+
```

Comment:
Is this allowed to grow forever? Is that a problem? Perhaps the storage is so small it doesn't matter?

Reply:
I'm going to go with that for now instead of bringing in a bounded Cache instance. None of the classes I saw seemed simple and straightforward